cirq/ops/three_qubit_gates_test.py

```diff
@@ -19,6 +19,7 @@
 
 import cirq
 
+FIRST_EIGHT_PRIMES = list(sympy.primerange(0, 20))
```

Comment:
Haha, that's a bit much. Just hardcode this list in each method, to keep them as concrete and decoupled as possible.

Reply:
got it, done.